src/main/java/com/dev/spring/dto/UserResponseDto.java

```diff
@@ -0,0 +1,43 @@
+package com.dev.spring.dto;
+
+import com.dev.spring.model.User;
+
+public class UserResponseDto {
+    private Long id;
+    private String email;
+    private String password;
+
+    public UserResponseDto() {
+
+    }
+
+    public UserResponseDto(User user) {
+        this.id = user.getId();
+        this.email = user.getEmail();
+        this.password = user.getPassword();
```

Comment:
let's not pass model(User) to constructor. It will create redundant dependency .